tests/integration/clientv3/connectivity/network_partition_test.go

```diff
@@ -23,23 +23,29 @@ import (
 	"testing"
 	"time"
 
+	"google.golang.org/grpc"
```

Comment:
I really wonder where this import inconsistency comes from, are you guys using a different gofmt? I'm on `go version go1.18.2 linux/amd64`

Reply:
Etcd haven't upgraded go to 1.18